src/mudlet.h

```diff
@@ -45,6 +45,7 @@
 #include <QSettings>
 #include <QTextOption>
 #include <QTime>
+#include <QTimer>
```

Comment:
We only have pointers to `QTimers` in this file so the forward reference to `class QTimer;` is sufficient I believe, and then this line is only needed in the class implementation file, a.k.a. `mudlet.cpp` which means a free recompilation of *every* class that uses this file is not needed - although it negates my previous comment. :wink:

Reply:
Fixed